website/docs/cdktf/create-and-deploy/terraform-cloud.mdx

```diff
@@ -24,27 +24,62 @@ Terraform uses persisted state data to keep track of the real-world resources it
 
-Workspaces can have one of three workflows that determine how Terraform Cloud interacts with your CDKTF application: CLI-driven, Version control, or API-driven (a more advanced option).
+Workspaces can have one of three workflows that determine how Terraform Cloud / Enterprise interacts with your CDKTF application: CLI-driven, Version control, or API-driven (a more advanced option).
 
 ### CLI-driven Workflow
 
-You can configure the Terraform Cloud [CLI-driven workflow](/cloud-docs/run/cli) for both new and existing CDKTF projects. This workflow lets you run CDKTF CLI commands to deploy your application and run Terraform operations remotely on Terraform Cloud.
+You can configure the Terraform Cloud [CLI-driven workflow](/cloud-docs/run/cli) for both new and existing CDKTF projects. This workflow lets you run CDKTF CLI commands to deploy your application and run Terraform operations remotely on Terraform Cloud / Enterprise.
 
 #### Connect New CDKTF Projects
 
-For new projects, do the following:
+First use `cdktf login` to log into Terraform Cloud or your Terraform Enterprise instance:
 
-1. Run `cdktf init` without passing the `--local` flag. This creates a new template CDKTF project for your chosen programming language. Templates generate a new application with the necessary file structure for you to start defining infrastructure.
-1. Choose a project language and provide a project name and description.
-1. Log in to Terraform Cloud. CDKTF uses saved credentials or asks you to log in.
-1. Select a Terraform Cloud organization and a name for your workspace.
-1. Choose whether to send crash reports to the CDKTF team.
+<Tabs>
+  <Tab heading="Terraform Cloud" group="tfc">
+
+    This command will take you through an interactive session to log in.
+
+    ```shell-session
+    $ cdktf login
+    ```
+
+    If you already have a Terraform Cloud API token, you can pass it into stdin:
+
+    ```shell-session
+    cat my-token.txt | cdktf login
+    ```
+
+  </Tab>
+
+  <Tab heading="Terraform Enterprise" group="tfe">
+
+    This command will take you through an interactive session to log in.
+
+    ```shell-session
+    $ cdktf login --tfe-hostname tfe.my-company.com
+    ```
+
+    If you already have a Terraform Enterprise API token, you can pass it into stdin:
+
+    ```shell-session
+    cat my-token.txt | cdktf login --tfe-hostname tfe.my-company.com
+    ```
+
+  </Tab>
+</Tabs>
+
+Next, run `cdktf init` without passing the `--local` flag.
+This creates a new template CDKTF project for your chosen programming language.
+It will be the basis for a new application with the necessary file structure for you to start defining infrastructure.
+Choose a project language and provide a project name and description.
+Select a Terraform Cloud organization and a name for your workspace and let CDKTF CLI create it for you.
+Next you can choose whether to send crash reports to the CDKTF team and select the providers you want to use.
```

Comment:
Reads like this could become an ordered list instead of just separate sentences with no connecting words?

Reply:
I removed the connecting words. I would love to have this a list, but this does not work with the tabs component inbetween